subscription-engine/kafka-ws/app/Kafka/KafkaConsumer.js

```diff
@@ -38,6 +40,13 @@ class KafkaConsumer {
     }
   }
 
+  /**
+   * Get kafka Status for internal uses
+   */
+  getKafkaStatus() {
+    return this.kafkaAvailable;
+  }
```

Comment:
Have you considered using the `getter` syntax?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get